docs/using.md

```diff
@@ -424,6 +424,30 @@ spec:
 
 1. To run a `Task`, create a new `TaskRun` which defines all inputs, outputs
    that the `Task` needs to run.
+
+  * Input and output resources can be embedded without defining Pipeline Resource. TaskRun resource can include either Pipeline Resource reference or resource spec but not both. For example
```

Comment:
maybe we can have an example of the case where we use a reference as well? (i think looking at these docs, folks might assume that embedding the spec is the default)